test/runtests.jl

```diff
@@ -856,13 +856,13 @@ end
                 """)
 
                 testaln("""
-                 ACGT  
+                 ACGT
                 AACGTTT
                  ^^^^
                 """)
 
                 testaln("""
-                  AC-GT  
```

Comment:
Also here, the two blank spaces have been removed, which I think might be making tests fail.